### PR TITLE
Refactor: rename modules

### LINKED ISSUE #15: Rename packages, repository please
### Packages
@plainsheet/plain-bottom-sheet-core -> @plainsheet/core 
@plainsheet/react-plain-bottom-sheet-> @plainsheet/react

### Repository
plainsheet/plain-bottom-sheet -> plainsheet/plainsheet

### PR DESCRIPTION
**Pull request by peter-byun**
## Why?
To simplify module names and keep its consistency across packages

## Description
resolves #15 issues/15

### CODE REVIEW

File: apps/pbs-docs/src/pages/apis/react.mdx [hunk 1]
```diff
@@ -1,38 +1,38 @@
-## `ReactPlainBottomSheet`
+## `BottomSheet`
 
 ### Interface
 
 ```ts
-ReactPlainBottomSheet: (props: PlainBottomSheetProps) => ReactNode;
+BottomSheet: (props: PlainBottomSheetProps) => ReactNode;
```
Comment (manudeli) [inline apps/pbs-docs/src/pages/apis/react.mdx:6]: PlainBottomSheeyProps -> BottomSheetProps

File: packages/react/src/BottomSheet.tsx [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+} from "react";
+import { createPortal } from "react-dom";
+
+export type PlainBottomSheetProps = {
+  children: ReactNode;
+  isOpen: boolean;
+  setIsOpen: (isOpen: boolean) => void;
+  mountingPoint?: Element | null;
+} & CoreProps;
+type CoreProps = OmitKeyof<BottomSheetProps, "content", "safely">;
```
Comment (manudeli) [inline packages/react/src/BottomSheet.tsx:27]: Remove safely please

File: packages/react/src/index.ts [hunk 3]
```diff
@@ -1 +1 @@
-export * from "./ReactPlainBottomSheet";
+export * from "./BottomSheet";
```
Comment (manudeli) [inline packages/react/src/index.ts:1]: Re-export not all exactly please. Public apis should be exported carefully

File: packages/utility/package.json [hunk 4]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "pbs-utility",
+  "name": "@plainsheet/utility",
   "version": "0.0.2",
```
Comment (manudeli) [inline packages/utility/package.json:3]: How about merging this package into just @plainsheet/core

File: packages/react/src/index.ts (hunk 3 above)
Comment (peter-byun) [reply packages/react/src/index.ts:1]: Thanks! Made an issue for this:
https://github.com/orgs/plainsheet/projects/2/views/1?pane=issue&itemId=75153379

File: packages/react/src/BottomSheet.tsx (hunk 2 above)
Comment (peter-byun) [reply packages/react/src/BottomSheet.tsx:27]: Caught some type errors thanks to this. Thank you.

File: apps/pbs-docs/src/pages/apis/react.mdx (hunk 1 above)
Comment (peter-byun) [reply apps/pbs-docs/src/pages/apis/react.mdx:6]: fixed at 8ba27281581820ea8f0c884a7215e09dcc30c871
Thank you.

File: packages/utility/package.json (hunk 4 above)
Comment (peter-byun) [reply packages/utility/package.json:3]: `@plainsheet/react` and `@plainsheet/core` share some chunks of utility functions, so I made it a separate package.

**Pull request merged**